scrapi/harvesters/osf.py

```diff
@@ -94,9 +94,10 @@ def schema(self):
             )
         }
 
-    def harvest(self, days_back=1):
-        start_date = str(date.today() - timedelta(int(days_back)))
-        end_date = str(date.today())
+    def harvest(self, start_date=None, end_date=None):
+
+        start_date = start_date or date.today().isoformat()
+        end_date = end_date or (date.today() - timedelta(1)).isoformat()
```

Comment:
This will make start and end date either a string or a datetime object, depending one whether it is called with arguments.